src/EventsHeader/DetailsSection.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DetailsSection } from './DetailsSection';
+import renderer from 'react-test-renderer';
+
+const t = () => '';
```

Comment:
What does this represent? 😁

Reply:
This is how you can mock `t` (the translate function in react-i18next) now this is a Stateless Functional Component.

Reply:
Ah good to know. Maybe add a comment about this?

Reply:
Done!